packages/extract/src/Extract.ts

```diff
@@ -182,7 +191,9 @@ export class Extract implements IRendererPlugin
      *  to convert. If left empty will use the main renderer
      * @return - One-dimensional array containing the pixel data of the entire texture
      */
-    public pixels(target?: DisplayObject|RenderTexture): Uint8Array
+
+    // any { x: double; y: double; scale : double; Width:double;height :double}
```

Comment:
Could you please remove these comments and new line. It's breaks the documentation which needs to immediately precede the method.
```suggestion
```